os/arch/arm/src/common/up_vfork.c

```diff
@@ -182,6 +186,13 @@ pid_t up_vfork(const struct vfork_s *context)
 		return (pid_t)ERROR;
 	}
 
+#ifdef CONFIG_DEBUG_MM_HEAPINFO
+	/* Update the pid information in stack node */
+	heapinfo_set_stack_node(child->cmn.stack_alloc_ptr, child->cmn.pid);
+	/* Subtract the stacksize from parent */
+	heapinfo_exclude_stacksize(child->cmn.stack_alloc_ptr);
```

Comment:
I think the order of these two function need to be changed.
If heapinfo_set_stack_node() is called first, heapinfo_exclude_stacksize() cannot access to parent. (because pid is updated to -1)

1. heapinfo_exclude_stacksize()
2. heapinfo_set_stack_node()

Reply:
Thanks~ modified.